README.md

```diff
@@ -73,6 +73,9 @@ if (cordova.platformId == 'android') {
 }
 ```
 
+### Browser Support
```

Comment:
May I suggest renaming this heading to "Browser Quirks", to align with the other section headings (i.e. "Android Quirks").